src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -68,142 +88,165 @@ const imageWidthSizes = {
+                          }}
+                        >
+                          {is_followed ? "Following" : "Follow"}
                         </Sans>
-                        <Flex flexWrap={isColumnLayout ? "wrap" : "nowrap"}>
-                          {featuredArtists}
-                        </Flex>
-                      </Box>
-                    )}
-                  </DescriptionContainer>
-                  <Spacer mb={1} />
-                </Box>
-              </Flex>
-              <Spacer mb={2} />
-            </header>
+                      )
+                    }}
+                  />
+                }
+              />
+            </EntityContainer>
           )
-        }}
-      </Responsive>
-    )
-  }
+        })
+
+        return (
+          <header>
+            <Flex flexDirection="column">
+              <Box>
+                <Background
+                  p={2}
+                  mt={[0, 3]}
+                  mb={3}
+                  headerImageUrl={resize(collection.headerImage, {
+                    width: imageWidth * (xs ? 2 : 1),
+                    height: imageHeight * (xs ? 2 : 1),
+                    quality: 80,
+                  })}
+                  height={imageHeight}
+                >
+                  <Overlay />
+                  {collection.credit && (
+                    <ImageCaption
+                      size={size}
+                      dangerouslySetInnerHTML={{ __html: collection.credit }}
+                    />
+                  )}
+                </Background>
+                <MetaContainer mb={2}>
+                  <BreadcrumbContainer size={["2", "3"]}>
+                    <a href="/collect">All works</a> /{" "}
+                    <a href={categoryTarget}>{collection.category}</a>
+                  </BreadcrumbContainer>
+                  <Spacer mt={1} />
+                  <Serif size={["6", "10"]}>{collection.title}</Serif>
+                </MetaContainer>
+                <Grid>
+                  <Row>
+                    <Col xs="12" sm="12" md="8" lg="8" xl="8">
+                      <Flex>
+                        <ExtendedSerif size="3">
+                          {smallerScreen ? (
+                            <ReadMore
+                              onReadMoreClicked={trackReadMoreClick}
+                              maxChars={chars}
+                              content={getReadMoreContent(
+                                collection.description
+                              )}
+                            />
+                          ) : (
+                            getReadMoreContent(collection.description)
+                          )}
+                        </ExtendedSerif>
+                      </Flex>
+                    </Col>
+                    <Col
+                      xs={12}
+                      sm={12}
+                      md={isColumnLayout ? "12" : "3"}
```

Comment:
@xtina-starr  what @damassi is mentioning here is you can omit `lg` and `xl`. 
Because that width applies to `md` and any breakpoint above it.

Not sure for `mdOffset` but it should behave the same I believe.

So

```typescript
<Col
  sm={12}
  md={isColumnLayout ? "12" : "3"}
>
  ...
</Col>
```

Reply:
From [Bootstrap docs](https://getbootstrap.com/docs/4.1/layout/grid/) which our grid library ([`styled-bootstrap-grid`](https://github.com/dragma/styled-bootstrap-grid)) is based on:

> Grid breakpoints are based on minimum width media queries, meaning they apply to that one breakpoint and all those above it (e.g., .col-sm-4 applies to small, medium, large, and extra large devices, but not the first xs breakpoint).

Reply:
ahhh I see!